src/test/java/webdriverFramework/SSMission.java

```diff
@@ -1,53 +1,211 @@
 package webdriverFramework;
 
 
-import org.openqa.selenium.By;
-import org.openqa.selenium.Keys;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import enums.Language;
+import com.google.common.base.Function;
+import enums.ValueForDealTypeDropdown;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.openqa.selenium.*;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
-import org.openqa.selenium.remote.server.handler.FindElement;
-import org.openqa.selenium.support.ui.ExpectedCondition;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.Select;
-import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.support.ui.*;
+import pageObjects.*;
 
 import java.io.File;
+import java.util.*;
+
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
 
-/**
- * Created by roman.pipchenko on 1/20/2017.
- */
 public class SSMission {
-    public static void main(String[] args) {
 
-        File file = new File("C:\\tools\\Firefox\\", "firefox" + ".exe");
-        if (file.exists()) {
-            System.setProperty("webdriver.firefox.bin", "C:\\tools\\Firefox\\firefox.exe");
-        }
+    static WebDriver driver;
+
+    public static String browser = "chrome";
 
-        System.setProperty("webdriver.gecko.driver", "src\\test\\resources\\geckodriver.exe");
-        WebDriver driver = new FirefoxDriver();
 
+    @Rule
+    public TestRule screenshotTaker = new ScreenshotTaker((TakesScreenshot) driver);
+
+    @BeforeClass
+    public static void setup() {
+        if (browser.equals("chrome")) {
+
+            File file = new File("C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe");
+            if (file.exists()) {
+                System.setProperty("webdriver.chrome.driver", "src\\test\\resources\\chromedriver.exe");
+                driver = new ChromeDriver();
+            }
+        } else {
+            File file = new File("C:\\tools\\Firefox\\", "firefox" + ".exe");
+            if (file.exists()) {
+                System.setProperty("webdriver.firefox.bin", "C:\\tools\\Firefox\\firefox.exe");
+            }
+            System.setProperty("webdriver.gecko.driver", "src\\test\\resources\\geckodriver.exe");
+            driver = new FirefoxDriver();
+        }
+    }
 
-        driver.get("https://www.ss.lv");
-        driver.findElement(By.xpath("//*[@href='/ru/']")).click();
+    @Test
+    public void testSS() throws InterruptedException {
         WebDriverWait wait = new WebDriverWait(driver, 5);
+        driver.get("https://www.ss.lv/lv");
+        driver.manage().window().maximize();
+
+        HomePage homePage = new HomePage(driver);
```

Comment:
If you put driver to static context - you won't need to create beans in test. When with MockitoJUnitRunner this might be moved to injected field.